src/app/metadata/features/metadata-browser/metadata-browser.component.ts

```diff
@@ -156,6 +158,7 @@ export class MetadataBrowserComponent implements OnInit {
   clearSearchQuery(): void {
     this.searchTerm = '';
     this.searchFormControl.setValue('');
+    this.#skip = 0;
     this.#performSearch();
```

Comment:
if (this.searchTerm) {
      this.searchTerm = '';
      this.searchFormControl.setValue('');
      this.#skip = DEFAULT_SKIP_VALUE;
      this.#performSearch();
    }